### PR TITLE
Fix footer links

### PR DESCRIPTION
**Pull request by jfoxworth**
## Description

This pr makes the links in the footer match the links in the main page. It does not address the font of the links.

## Testing and QA

- Download the branch
- npm run start
- go to http://localhost:8080/
- Open dallasnews.com in separate window
- Compare the footers and make sure that the link lists match

### CODE REVIEW

**Comment by rlsmileg** [comment]
Company link is wrong. should be "https://www.dallasnewscorporation.com/about/" @jfoxworth

**Comment by rlsmileg** [comment]
Website Support link / href is wrong. Should be "https://www.dallasnews.com/help/" Last column of links. @jfoxworth

**Comment by jfoxworth** [comment]
Links should be fixed

File: src/footer/footer.html [hunk 1]
```diff
@@ -240,7 +240,7 @@
             <li
               class="cursor-pointer text-gray-dark hover_underline pb-6 lg_pr-0 pr-7 inline-block lg_block"
             >
-              <a href="https://www.dallasnews.com/help/about-us/"
```
Comment (rlsmileg) [inline src/footer/footer.html]: This link was correct. Change it back. @jfoxworth

File: src/footer/footer.html [hunk 2]
```diff
@@ -1,230 +1,390 @@
...
+            <li
+              class="cursor-pointer text-gray-dark hover_underline pb-6 lg_pr-0 pr-7 inline-block lg_block"
+            >
+              <a href="https://finddallasnews.com/">Today’s Paper</a>
+            </li>
+          </ul>
+        </div>
+        <div class="column small-12 medium-12 large-expand">
+          <div class="font-medium block uppercase hover_underline pb-6">
+            <a href="http://dmnmedia.com/about-us/company-info">Company</a>
```
Comment (rlsmileg) [inline src/footer/footer.html:235]: This link is wrong. Should be... https://www.dallasnewscorporation.com/about/

**Review by rlsmileg — CHANGES_REQUESTED** [review]
@jfoxworth

**Review by rlsmileg — APPROVED** [review]
Looks good

**Pull request merged**